### PR TITLE
Add methods add, set for Group Analytics. Only single values supported

### LINKED ISSUE #216: Missing set group method
This React Native SDK is missing the functionality to set a user's group analytics. This missing method should accept these two parameters: _group key_ and _group name_ 

### Reference
*  Mixpanel Help Center [Group Analytics](https://help.mixpanel.com/hc/en-us/articles/360025333632)
* [iOS SDK docs](https://developer.mixpanel.com/docs/ios#section-group-analytics)
* [Browser JS SDK docs](https://developer.mixpanel.com/docs/javascript-full-api-reference#section-mixpanel-set-group)

### PR DESCRIPTION
**Pull request by danactive**
Closed #216 

Tested with this React Native code in Android/iOS simulators
```
      await this.mixpanel.addGroup('company', 'mixpanel');
```
```
      await this.mixpanel.setGroup('company', 'mixpanel');
```

Mixpanel admin screen
Mixpanel admin screen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,6 +161,16 @@ Mixpanel.set({"$email": "elvis@email.com"});
 // Set People Properties Once (warning: if no mixpanel profile has been assigned to the current user when this method is called, it will automatically create a new mixpanel profile and the user will no longer be anonymous in Mixpanel)
 Mixpanel.setOnce({"$email": "elvis@email.com", "Created": new Date().toISOString()});
 
+// Add a new Group for this user.
+// @param Group key
+// @param A valid Mixpanel property type
+Mixpanel.setGroup('company', 'mixpanel');
```
Comment (danactive) [inline README.md]:
```suggestion
Mixpanel.addGroup('company', 'mixpanel');
```

Comment (dwilt) [reply README.md:167]: @davodesign84 why wasn't this change added? The README is currently incorrect

**Comment by dwilt** [comment]
@danactive curious, why did you have the `setGroup` 2nd param be a string and not an array of strings like the [JS implementation](https://developer.mixpanel.com/docs/javascript#adding-users-to-a-group) does?

**Comment by danactive** [comment]
@dwilt My use-case only needed a single value and my deadline was tight so I did the minimum to get my job done. My design should not block another developer to add the multiple values

**Pull request merged**